client/state/data-layer/wpcom/sites/stats/google-my-business/test/index.js

```diff
@@ -0,0 +1,97 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import { GOOGLE_MY_BUSINESS_STATS_SEARCH_REQUEST } from 'state/action-types';
+import { fetchGoogleMyBusinessStats, receiveStats } from '../';
+import { receiveGoogleMyBusinessStats } from 'state/google-my-business/actions';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+describe( '#fetchGoogleMyBusinessStatsSearch', () => {
+	test( 'should dispatch HTTP request to Google My Business stats endpoint', () => {
+		const action = {
+			type: GOOGLE_MY_BUSINESS_STATS_SEARCH_REQUEST,
+			interval: 'week',
+			aggregation: 'total',
+			siteId: 12345,
+			statType: 'searches',
+		};
+		const dispatch = sinon.spy();
+
+		fetchGoogleMyBusinessStats( { dispatch }, action );
+
+		expect( dispatch ).to.have.been.calledOnce;
+		expect( dispatch ).to.have.been.calledWith(
+			http(
+				{
+					path: '/sites/12345/stats/google-my-business/searches',
```

Comment:
We're still referencing a wrong stat type here - this should be `queries` instead of `searches`.